src/Extensions/UserDefinedFormControllerExtension.php

```diff
@@ -53,44 +55,84 @@ public function updateEmail(Email $email, EmailRecipient $recipient): void
                 ->filter('EmailAddress', $recipient->EmailAddress)->first();
 
             if ($encryptionCertificateEntry && $encryptionCertificateEntry->EncryptionCrt->exists()) {
-                $pathToFile = $this->getFilePath($encryptionCertificateEntry->EncryptionCrt);
+                $encryptionPathFile = $this->getFilePath($encryptionCertificateEntry->EncryptionCrt);
```

Comment:
I assume keeping this as a `.crt` file extension is ok given that the `construct` of the `SMimeEncrypter` requests for a `X.509 certificate` which I assume is a `.crt` file?
![Screen Shot 2024-06-11 at 10 51 47 AM](https://github.com/silverstripeltd/user-configurable-smime-forms/assets/27745093/7d2cb9d8-ed5b-4d9c-94d2-e5bfe99f6eb4)